sources/packages/web/src/assets/css/vuetify.scss

```diff
@@ -131,6 +131,12 @@
   border-bottom: 2px solid $common-border-color;
 }
 
+// CSS to match the height of progress bar as per figma.
+.v-slider.v-input--horizontal .v-slider-track__fill,
```

Comment:
Is this assuming that every progress bar across the AEST/student/etc. will be the same?

Reply:
It was done assuming the stepper progress bar height is same through the application. But can confirm the same with @hellolynn-tbtb

Reply:
"stepper progress bar height" can maybe be the same but here we are setting the height for any v-slider, right?
My point was that maybe we should do it by adding one more class to the CSS filter.